lib/AST/AvailabilityDomain.cpp

```diff
@@ -139,6 +145,7 @@ bool AvailabilityDomain::isActive(const ASTContext &ctx) const {
   switch (getKind()) {
   case Kind::Universal:
   case Kind::SwiftLanguage:
+  case Kind::SwiftToolchain:
```

Comment:
I think this should be false so that we don't emit any diagnostics about `_SwiftToolchain` availability.